### PR TITLE
Documentation of USER MANAGEMENT

### PR DESCRIPTION
**Pull request by Lojjs**
Based on top of https://github.com/neo4j/neo4j-documentation/pull/509

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala [hunk 1]
```diff
@@ -59,7 +59,7 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
         resultTable()
       }
     }
-    section("Database administration", "administration-security-administration-database-privileges", "enterprise-edition") {
+    section("Database administration", administration-security-administration-database-prive"", "enterprise-edition") {
```
Comment (Hunterness) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala]: ?

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala [hunk 2]
```diff
@@ -245,11 +245,10 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
         query("DENY ACCESS ON DATABASE * TO usermanager", ResultAssertions((r) => {
           assertStats(r, systemUpdates = 1)
         })) {
-          statsOnlyResultTable()
-        }
-        p("And DENY START and STOP for normal databases:")
+          statsOnlyResultTab("And DENY START and STOP for normal databases:")
         query("DENY START ON DATABASE * TO usermanager", ResultAssertions((r) => {
           assertStats(r, systemUpdates = 1)
+        })) {
```
Comment (Hunterness) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala]: ?

**Review by Hunterness — CHANGES_REQUESTED** [review]
I can't build and look at it but from just reviewing the code most looks good. 

Have two places I don't understand what you are trying to accomplish (might be something that should not have been changed) that I want you to look at before I approve :)

File: cypher/cypher-docs/src/docs/dev/ql/administration/security/dbms/admin-role-dbms.asciidoc [hunk 3]
```diff
@@ -4,10 +4,10 @@ These include:
 * Create or drop databases
 * Change configuration parameters
 * Manage transactions
-* Manage users and roles (role management by itself is assignable using <<administration-security-administration-dbms-privileges-role-management, Cypher commands>>)
+* Manage users and roles
```
Comment (Hunterness) [inline cypher/cypher-docs/src/docs/dev/ql/administration/security/dbms/admin-role-dbms.asciidoc:7]: The description of this list is `the `admin` role has a number of built-in privileges that cannot be assigned using Cypher commands. These include:` so maybe the description  should be changed?

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala (hunk 1 above)
Comment (Lojjs) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala]: Not intentional, will fix

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala (hunk 2 above)
Comment (Lojjs) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala]: Oops this looks wrong

File: cypher/cypher-docs/src/docs/dev/ql/administration/security/dbms/admin-role-dbms.asciidoc (hunk 3 above)
Comment (Lojjs) [reply cypher/cypher-docs/src/docs/dev/ql/administration/security/dbms/admin-role-dbms.asciidoc:7]: Good point

**Review by Hunterness — APPROVED** [review]

**Comment by Lojjs** [comment]
Included as part of #509

**Pull request closed without merging**